treesheets/treesheets.vcxproj

```diff
@@ -66,7 +66,8 @@
       <WarningLevel>Level3</WarningLevel>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
       <DisableSpecificWarnings>4127;4512;4201;4146;4456;4702</DisableSpecificWarnings>
-      <AdditionalOptions>/std:c++latest %(AdditionalOptions)</AdditionalOptions>
+      <AdditionalOptions>/std:c++17 %(AdditionalOptions)</AdditionalOptions>
```

Comment:
I don't believe we need the `c++17` flag here, just the `BuildStlModules` is sufficient?

Reply:
I tried this only with `BuildStlModules` but that did not help because the compiler complaints about the ambiguity of the == operator in TreeSheets and wxWidgets.

Reply:
Ok, I'd need to see that error if that can be solved otherwise.. Lobster may start relying on C++20 features and then this will break also.

Reply:
This is the error message with `/std:c++latest` but with STL modules deactivated:
```
Run msbuild.exe TreeSheets\treesheets.sln /p:Configuration=Release /p:Platform=x64
MSBuild version 17.6.3+07e294721 for .NET Framework
Building the projects in this solution one at a time. To enable parallel build, please add the "-m" switch.
Build started 6/21/2023 6:55:16 PM.

Project "D:\a\treesheets\treesheets\TreeSheets\treesheets.sln" on node 1 (default targets).
ValidateSolutionConfiguration:
  Building solution configuration "Release|x64".
Project "D:\a\treesheets\treesheets\TreeSheets\treesheets.sln" (1) is building "D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj.metaproj" (2) on node 1 (default targets).
Project "D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj.metaproj" (2) is building "D:\a\treesheets\treesheets\lobster\lobster\language.vcxproj" (3) on node 1 (default targets).
PrepareForBuild:
  Creating directory "..\..\build\treesheets\language\x64\Release\".
  Creating directory "..\..\build\treesheets\language\x64\Release\language.tlog\".
InitializeBuildStatus:
  Creating "..\..\build\treesheets\language\x64\Release\language.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
ClCompile:
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\bin\HostX[8](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:9)6\x64\CL.exe /c /I..\src /I..\include /I..\external\freetype\include /I..\external\SDL\include /I..\external\libtcc /I..\external\tracy /I\Include /Zi /nologo /W4 /WX- /diagnostics:column /MP /O2 /Ob2 /Oi /Ot /GL /D WIN32 /D NDEBUG /D _CONSOLE /D SDL_XAUDIO2_HAS_SDK=1 /D BUILD_CONTEXT_treesheets /D _MBCS /Gm- /EHsc /MT /GS- /Gy /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /std:c++latest /permissive- /Yc"lobster/stdafx.h" /Fp"..\..\build\treesheets\language\x64\Release\language64.pch" /Fo"..\..\build\treesheets\language\x64\Release\\" /Fd"..\..\build\treesheets\language\language64.pdb" /external:W4 /Gd /TP /wd4127 /wd4512 /wd4201 /wd4146 /wd4456 /wd4702 /FC /errorReport:queue ..\src\stdafx.cpp
  stdafx.cpp
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\bin\HostX86\x64\CL.exe /c /I..\src /I..\include /I..\external\freetype\include /I..\external\SDL\include /I..\external\libtcc /I..\external\tracy /I\Include /Zi /nologo /W4 /WX- /diagnostics:column /MP /O2 /Ob2 /Oi /Ot /GL /D WIN32 /D NDEBUG /D _CONSOLE /D SDL_XAUDIO2_HAS_SDK=1 /D BUILD_CONTEXT_treesheets /D _MBCS /Gm- /EHsc /MT /GS- /Gy /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /std:c++latest /permissive- /Fo"..\..\build\treesheets\language\x64\Release\\" /Fd"..\..\build\treesheets\language\language64.pdb" /external:W4 /Gd /TP /wd4127 /wd4512 /wd4201 /wd4146 /wd4456 /wd4702 /FC /errorReport:queue ..\external\flatbuffers\src\idl_gen_text.cpp ..\external\flatbuffers\src\idl_parser.cpp ..\external\flatbuffers\src\util.cpp
  idl_gen_text.cpp
  idl_parser.cpp
  util.cpp
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\bin\HostX86\x64\CL.exe /c /I..\src /I..\include /I..\external\freetype\include /I..\external\SDL\include /I..\external\libtcc /I..\external\tracy /I\Include /Zi /nologo /W3 /WX- /diagnostics:column /MP /O2 /Ob2 /Oi /Ot /GL /D WIN32 /D NDEBUG /D _CONSOLE /D SDL_XAUDIO2_HAS_SDK=1 /D BUILD_CONTEXT_treesheets /D _MBCS /Gm- /EHsc /MT /GS- /Gy /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /permissive- /Fo"..\..\build\treesheets\language\x64\Release\\" /Fd"..\..\build\treesheets\language\language64.pdb" /external:W3 /Gd /TC /wd4127 /wd4512 /wd4201 /wd4146 /wd4456 /wd4702 /FC /errorReport:queue ..\external\libtcc\libtcc.c
  libtcc.c
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\bin\HostX86\x64\CL.exe /c /I..\src /I..\include /I..\external\freetype\include /I..\external\SDL\include /I..\external\libtcc /I..\external\tracy /I\Include /Zi /nologo /W4 /WX- /diagnostics:column /MP /O2 /Ob2 /Oi /Ot /GL /D WIN32 /D NDEBUG /D _CONSOLE /D SDL_XAUDIO2_HAS_SDK=1 /D BUILD_CONTEXT_treesheets /D _MBCS /Gm- /EHsc /MT /GS- /Gy /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /std:c++latest /permissive- /Yu"lobster/stdafx.h" /Fp"..\..\build\treesheets\language\x64\Release\language64.pch" /Fo"..\..\build\treesheets\language\x64\Release\\" /Fd"..\..\build\treesheets\language\language64.pdb" /external:W4 /Gd /TP /wd4127 /wd4512 /wd4201 /wd4146 /wd4456 /wd4702 /FC /errorReport:queue ..\src\builtins.cpp ..\src\disasm.cpp ..\src\file.cpp ..\src\compiler.cpp ..\src\lobsterreader.cpp ..\src\platform.cpp ..\src\tccbind.cpp ..\src\tocpp.cpp ..\src\vm.cpp ..\src\vmdata.cpp
  builtins.cpp
  disasm.cpp
  file.cpp
  compiler.cpp
  lobsterreader.cpp
  platform.cpp
  tccbind.cpp
  tocpp.cpp
  vm.cpp
  vmdata.cpp
Lib:
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\bin\HostX86\x64\Lib.exe /OUT:"..\..\build\treesheets\language\language64.lib" /NOLOGO /MACHINE:X64 /LTCG ..\..\build\treesheets\language\x64\Release\idl_gen_text.obj
  ..\..\build\treesheets\language\x64\Release\idl_parser.obj
  ..\..\build\treesheets\language\x64\Release\util.obj
  ..\..\build\treesheets\language\x64\Release\libtcc.obj
  ..\..\build\treesheets\language\x64\Release\builtins.obj
  ..\..\build\treesheets\language\x64\Release\disasm.obj
  ..\..\build\treesheets\language\x64\Release\file.obj
  ..\..\build\treesheets\language\x64\Release\compiler.obj
  ..\..\build\treesheets\language\x64\Release\lobsterreader.obj
  ..\..\build\treesheets\language\x64\Release\platform.obj
  ..\..\build\treesheets\language\x64\Release\stdafx.obj
  ..\..\build\treesheets\language\x64\Release\tccbind.obj
  ..\..\build\treesheets\language\x64\Release\tocpp.obj
  ..\..\build\treesheets\language\x64\Release\vm.obj
  ..\..\build\treesheets\language\x64\Release\vmdata.obj
  language.vcxproj -> D:\a\treesheets\treesheets\build\treesheets\language\language64.lib
FinalizeBuildStatus:
  Deleting file "..\..\build\treesheets\language\x64\Release\language.tlog\unsuccessfulbuild".
  Touching "..\..\build\treesheets\language\x64\Release\language.tlog\language.lastbuildstate".
Done Building Project "D:\a\treesheets\treesheets\lobster\lobster\language.vcxproj" (default targets).
Project "D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj.metaproj" (2) is building "D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj" (4) on node 1 (default targets).
PrepareForBuild:
  Creating directory "x64\Release\".
  Creating directory "x64\Release\treesheets.tlog\".
InitializeBuildStatus:
  Creating "x64\Release\treesheets.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
ClCompile:
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\bin\HostX86\x64\CL.exe /c /I..\wxWidgets\include /I..\wxWidgets\lib\vc_x64_lib\mswu /Zi /nologo /W3 /WX- /diagnostics:column /O2 /D NDEBUG /D WINVER=0x0400 /D wxUSE_GUI=1 /D NDEBUG /D WINVER=0x0400 /D wxUSE_GUI=1 /D _MBCS /Gm- /EHsc /MT /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /Yc"stdafx.h" /Fp"x64\Release\TreeSheets.pch" /Fo"x64\Release\\" /Fd"x64\Release\vc143.pdb" /external:W3 /Gd /TP /wd4127 /wd4512 /wd4201 /wd4146 /wd4456 /wd4702 /FC /errorReport:queue /std:c++latest ..\src\stdafx.cpp
  stdafx.cpp
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\bin\HostX86\x64\CL.exe /c /I..\lobster\src /I..\lobster\include /Zi /nologo /W3 /WX- /diagnostics:column /O2 /D NDEBUG /D WINVER=0x0400 /D wxUSE_GUI=1 /D NDEBUG /D WINVER=0x0400 /D wxUSE_GUI=1 /D _MBCS /Gm- /EHsc /MT /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /Fo"x64\Release\\" /Fd"x64\Release\vc143.pdb" /external:W3 /Gd /TP /wd4127 /wd4512 /wd4201 /wd4146 /wd4456 /wd4702 /FC /errorReport:queue /std:c++latest ..\src\lobster_impl.cpp
  lobster_impl.cpp
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\bin\HostX86\x64\CL.exe /c /I..\wxWidgets\include /I..\wxWidgets\lib\vc_x64_lib\mswu /Zi /nologo /W3 /WX- /diagnostics:column /O2 /D NDEBUG /D WINVER=0x0400 /D wxUSE_GUI=1 /D NDEBUG /D WINVER=0x0400 /D wxUSE_GUI=1 /D _MBCS /Gm- /EHsc /MT /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /Yu"stdafx.h" /Fp"x64\Release\TreeSheets.pch" /Fo"x64\Release\\" /Fd"x64\Release\vc143.pdb" /external:W3 /Gd /TP /wd4127 /wd4512 /wd4201 /wd4146 /wd4456 /wd4702 /FC /errorReport:queue /std:c++latest ..\src\main.cpp
  main.cpp
D:\a\treesheets\treesheets\src\document.h(350,15): error C2666: 'treesheets::Selection::operator ==': overloaded functions have similar conversions [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\selection.h(34,10): message : could be 'bool treesheets::Selection::operator ==(const treesheets::Selection &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\pen.h(138,13): message : or       'bool operator ==(wxPenStyle,wxDeprecatedGUIConstants)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\brush.h(107,13): message : or       'bool operator ==(wxBrushStyle,wxDeprecatedGUIConstants)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(138,13): message : or       'bool operator ==(long,const wxWindowIDRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(133,13): message : or       'bool operator ==(int,const wxWindowIDRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(128,13): message : or       'bool operator ==(const wxWindowIDRef &,long)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(123,13): message : or       'bool operator ==(const wxWindowIDRef &,int)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(118,13): message : or       'bool operator ==(const wxWindowIDRef &,const wxWindowIDRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(688,13): message : or       'bool operator ==(wxFontWeight,wxDeprecatedGUIConstants)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(682,13): message : or       'bool operator ==(wxFontStyle,wxDeprecatedGUIConstants)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(676,13): message : or       'bool operator ==(wxFontFamily,wxDeprecatedGUIConstants)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\guiddef.h(1[9](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:10)2,15): message : or       'bool operator ==(const GUID &,const GUID &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(878,13): message : or       'bool operator ==(const wxRect &,const wxRect &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(602,13): message : or       'bool operator ==(const wxPoint &,const wxPoint &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(472,13): message : or       'bool operator ==(const wxRealPoint &,const wxRealPoint &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(350,13): message : or       'bool operator ==(const wxSize &,const wxSize &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\longlong.h([10](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:11)56,13): message : or       'bool operator ==(unsigned long,const wxULongLong &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\longlong.h(1043,13): message : or       'bool operator ==(long,const wxLongLong &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4084,1): message : or       'bool operator ==(const wxCStrData &,const char *)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4084,1): message : or       'bool operator ==(const char *,const wxCStrData &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4082,1): message : or       'bool operator ==(const wxCStrData &,const wchar_t *)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4082,1): message : or       'bool operator ==(const wchar_t *,const wxCStrData &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4035,13): message : or       'bool operator ==(const wxString &,wchar_t)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4034,13): message : or       'bool operator ==(const wxString &,char)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4033,13): message : or       'bool operator ==(const wxString &,const wxUniCharRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4032,13): message : or       'bool operator ==(const wxString &,const wxUniChar &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4031,13): message : or       'bool operator ==(int,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4030,13): message : or       'bool operator ==(wchar_t,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4029,13): message : or       'bool operator ==(char,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4028,13): message : or       'bool operator ==(const wxUniCharRef &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4027,13): message : or       'bool operator ==(const wxUniChar &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4006,13): message : or       'bool operator ==(const wxScopedCharBuffer &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4004,13): message : or       'bool operator ==(const wxString &,const wxScopedCharBuffer &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3996,13): message : or       'bool operator ==(const wxScopedWCharBuffer &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3994,13): message : or       'bool operator ==(const wxString &,const wxScopedWCharBuffer &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3987,13): message : or       'bool operator ==(const wxCStrData &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3985,13): message : or       'bool operator ==(const wxString &,const wxCStrData &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3972,13): message : or       'bool operator ==(const wxString &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3967,1): message : or       'bool operator ==(const wxString &,const char *)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3967,1): message : or       'bool operator ==(const char *,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3965,1): message : or       'bool operator ==(const wxString &,const wchar_t *)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3965,1): message : or       'bool operator ==(const wchar_t *,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(378,1): message : or       'bool operator ==(const wxUniChar &,const wxUniCharRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(376,1): message : or       'bool operator ==(wchar_t,const wxUniCharRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(375,1): message : or       'bool operator ==(wchar_t,const wxUniChar &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(373,1): message : or       'bool operator ==(char,const wxUniCharRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(372,1): message : or       'bool operator ==(char,const wxUniChar &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\thread(242,36): message : or       'bool std::operator ==(std::thread::id,std::thread::id) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\selection.h(34,10): message : or 'bool treesheets::Selection::operator ==(const treesheets::Selection &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(372,1): message : or 'bool operator ==(char,const wxUniChar &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(373,1): message : or 'bool operator ==(char,const wxUniCharRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(375,1): message : or 'bool operator ==(wchar_t,const wxUniChar &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(376,1): message : or 'bool operator ==(wchar_t,const wxUniCharRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(378,1): message : or 'bool operator ==(const wxUniChar &,const wxUniCharRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3965,1): message : or 'bool operator ==(const wchar_t *,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3965,1): message : or 'bool operator ==(const wxString &,const wchar_t *)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3967,1): message : or 'bool operator ==(const char *,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3967,1): message : or 'bool operator ==(const wxString &,const char *)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3972,13): message : or 'bool operator ==(const wxString &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3985,13): message : or 'bool operator ==(const wxString &,const wxCStrData &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3987,13): message : or 'bool operator ==(const wxCStrData &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3994,13): message : or 'bool operator ==(const wxString &,const wxScopedWCharBuffer &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3996,13): message : or 'bool operator ==(const wxScopedWCharBuffer &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4004,13): message : or 'bool operator ==(const wxString &,const wxScopedCharBuffer &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4006,13): message : or 'bool operator ==(const wxScopedCharBuffer &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4027,13): message : or 'bool operator ==(const wxUniChar &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4028,13): message : or 'bool operator ==(const wxUniCharRef &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4029,13): message : or 'bool operator ==(char,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4030,13): message : or 'bool operator ==(wchar_t,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4031,13): message : or 'bool operator ==(int,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4032,13): message : or 'bool operator ==(const wxString &,const wxUniChar &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4033,13): message : or 'bool operator ==(const wxString &,const wxUniCharRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4034,13): message : or 'bool operator ==(const wxString &,char)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4035,13): message : or 'bool operator ==(const wxString &,wchar_t)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4082,1): message : or 'bool operator ==(const wchar_t *,const wxCStrData &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4082,1): message : or 'bool operator ==(const wxCStrData &,const wchar_t *)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4084,1): message : or 'bool operator ==(const char *,const wxCStrData &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4084,1): message : or 'bool operator ==(const wxCStrData &,const char *)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\longlong.h(1043,13): message : or 'bool operator ==(long,const wxLongLong &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\longlong.h(1056,13): message : or 'bool operator ==(unsigned long,const wxULongLong &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(350,13): message : or 'bool operator ==(const wxSize &,const wxSize &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(472,13): message : or 'bool operator ==(const wxRealPoint &,const wxRealPoint &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(602,13): message : or 'bool operator ==(const wxPoint &,const wxPoint &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(878,13): message : or 'bool operator ==(const wxRect &,const wxRect &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\guiddef.h(192,15): message : or 'bool operator ==(const GUID &,const GUID &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(676,13): message : or 'bool operator ==(wxFontFamily,wxDeprecatedGUIConstants)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(682,13): message : or 'bool operator ==(wxFontStyle,wxDeprecatedGUIConstants)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(688,13): message : or 'bool operator ==(wxFontWeight,wxDeprecatedGUIConstants)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h([11](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:12)8,13): message : or 'bool operator ==(const wxWindowIDRef &,const wxWindowIDRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h([12](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:13)3,[13](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:14)): message : or 'bool operator ==(const wxWindowIDRef &,int)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(128,13): message : or 'bool operator ==(const wxWindowIDRef &,long)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(133,13): message : or 'bool operator ==(int,const wxWindowIDRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(138,13): message : or 'bool operator ==(long,const wxWindowIDRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\brush.h(107,13): message : or 'bool operator ==(wxBrushStyle,wxDeprecatedGUIConstants)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\pen.h(138,13): message : or 'bool operator ==(wxPenStyle,wxDeprecatedGUIConstants)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\[14](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:15).36.32532\include\thread(242,36): message : or 'bool std::operator ==(std::thread::id,std::thread::id) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\sharedptr.h([15](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:16)8,6): message : or       'bool operator ==(const wxSharedPtr<T> &,const wxSharedPtr<U> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool operator ==(const wxSharedPtr<T> &,const wxSharedPtr<U> &)': could not deduce template argument for 'const wxSharedPtr<T> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\forward_list(1570,17): message : or       'bool std::operator ==(const std::forward_list<_Ty,_Alloc> &,const std::forward_list<_Ty,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::forward_list<_Ty,_Alloc> &,const std::forward_list<_Ty,_Alloc> &)': could not deduce template argument for 'const std::forward_list<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\queue(181,17): message : or       'bool std::operator ==(const std::queue<_Ty,_Container> &,const std::queue<_Ty,_Container> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::queue<_Ty,_Container> &,const std::queue<_Ty,_Container> &)': could not deduce template argument for 'const std::queue<_Ty,_Container> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\deque(1703,17): message : or       'bool std::operator ==(const std::deque<_Ty,_Alloc> &,const std::deque<_Ty,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::deque<_Ty,_Alloc> &,const std::deque<_Ty,_Alloc> &)': could not deduce template argument for 'const std::deque<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\map(630,17): message : or       'bool std::operator ==(const std::multimap<_Kty,_Ty,_Pr,_Alloc> &,const std::multimap<_Kty,_Ty,_Pr,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::multimap<_Kty,_Ty,_Pr,_Alloc> &,const std::multimap<_Kty,_Ty,_Pr,_Alloc> &)': could not deduce template argument for 'const std::multimap<_Kty,_Ty,_Pr,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\map(399,17): message : or       'bool std::operator ==(const std::map<_Kty,_Ty,_Pr,_Alloc> &,const std::map<_Kty,_Ty,_Pr,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::map<_Kty,_Ty,_Pr,_Alloc> &,const std::map<_Kty,_Ty,_Pr,_Alloc> &)': could not deduce template argument for 'const std::map<_Kty,_Ty,_Pr,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\array(771,30): message : or       'bool std::operator ==(const std::array<_Ty,_Size> &,const std::array<_Ty,_Size> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::array<_Ty,_Size> &,const std::array<_Ty,_Size> &)': could not deduce template argument for 'const std::array<_Ty,_Size> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(3549,30): message : or       'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,std::nullptr_t) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::unique_ptr<_Ty,_Dx> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(3501,30): message : or       'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,const std::unique_ptr<_Ty2,_Dx2> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,const std::unique_ptr<_Ty2,_Dx2> &)': could not deduce template argument for 'const std::unique_ptr<_Ty,_Dx> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(1909,17): message : or       'bool std::operator ==(const std::shared_ptr<_Ty> &,std::nullptr_t) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::shared_ptr<_Ty> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::shared_ptr<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(1872,17): message : or       'bool std::operator ==(const std::shared_ptr<_Ty> &,const std::shared_ptr<_Ty0> &) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::shared_ptr<_Ty> &,const std::shared_ptr<_Ty0> &) noexcept': could not deduce template argument for 'const std::shared_ptr<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\iterator(489,17): message : or       'bool std::operator ==(const std::istreambuf_iterator<_Elem,_Traits> &,const std::istreambuf_iterator<_Elem,_Traits> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::istreambuf_iterator<_Elem,_Traits> &,const std::istreambuf_iterator<_Elem,_Traits> &)': could not deduce template argument for 'const std::istreambuf_iterator<_Elem,_Traits> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\iterator(307,17): message : or       'bool std::operator ==(const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &,const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &,const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &) noexcept': could not deduce template argument for 'const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(833,27): message : or       'bool std::operator ==(const _Ty1 &,const std::optional<_Ty2> &) noexcept(<expr>)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const _Ty1 &,const std::optional<_Ty2> &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty2> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(824,27): message : or       'bool std::operator ==(const std::optional<_Ty> &,const _Ty2 &) noexcept(<expr>)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::optional<_Ty> &,const _Ty2 &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(735,27): message : or       'bool std::operator ==(const std::optional<_Ty> &,std::nullopt_t) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::optional<_Ty> &,std::nullopt_t) noexcept': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(619,27): message : or       'bool std::operator ==(const std::optional<_Ty> &,const std::optional<_Ty2> &) noexcept(<expr>)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::optional<_Ty> &,const std::optional<_Ty2> &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\functional(1158,17): message : or       'bool std::operator ==(const std::function<_Fty> &,std::nullptr_t) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::function<_Fty> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::function<_Fty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\unordered_map(908,17): message : or       'bool std::operator ==(const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)': could not deduce template argument for 'const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\unordered_map(539,17): message : or       'bool std::operator ==(const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)': could not deduce template argument for 'const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\list(1873,17): message : or       'bool std::operator ==(const std::list<_Ty,_Alloc> &,const std::list<_Ty,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::list<_Ty,_Alloc> &,const std::list<_Ty,_Alloc> &)': could not deduce template argument for 'const std::list<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\vector(2219,30): message : or       'bool std::operator ==(const std::vector<_Ty,_Alloc> &,const std::vector<_Ty,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::vector<_Ty,_Alloc> &,const std::vector<_Ty,_Alloc> &)': could not deduce template argument for 'const std::vector<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(5051,30): message : or       'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const _Elem *const )' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const _Elem *const )': could not deduce template argument for 'const std::basic_string<_Elem,_Traits,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(5045,30): message : or       'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const std::basic_string<_Elem,_Traits,_Alloc> &) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const std::basic_string<_Elem,_Traits,_Alloc> &) noexcept': could not deduce template argument for 'const std::basic_string<_Elem,_Traits,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring([16](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:17)94,27): message : or       'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const _Identity<std::basic_string_view<_Elem,_Traits>>::type) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const _Identity<std::basic_string_view<_Elem,_Traits>>::type) noexcept': could not deduce template argument for 'const std::basic_string_view<_Elem,_Traits>' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(1680,27): message : or       'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const std::basic_string_view<_Elem,_Traits>) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const std::basic_string_view<_Elem,_Traits>) noexcept': could not deduce template argument for 'const std::basic_string_view<_Elem,_Traits>' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xmemory(1011,30): message : or       'bool std::operator ==(const std::allocator<_Ty> &,const std::allocator<_Other> &) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::allocator<_Ty> &,const std::allocator<_Other> &) noexcept': could not deduce template argument for 'const std::allocator<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\tuple(669,27): message : or       'bool std::operator ==(const std::tuple<_Types...> &,const std::tuple<_Types...> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::tuple<_Types...> &,const std::tuple<_Types...> &)': could not deduce template argument for 'const std::tuple<_Types...> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xutility(4162,5): message : or       'bool std::operator ==(const std::move_iterator<_Iter> &,const std::move_iterator<_Iter2> &) noexcept(<expr>)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::move_iterator<_Iter> &,const std::move_iterator<_Iter2> &) noexcept(<expr>)': could not deduce template argument for 'const std::move_iterator<_Iter> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xutility(1464,5): message : or       'bool std::operator ==(const std::reverse_iterator<_BidIt> &,const std::reverse_iterator<_BidIt2> &) noexcept(<expr>)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::reverse_iterator<_BidIt> &,const std::reverse_iterator<_BidIt2> &) noexcept(<expr>)': could not deduce template argument for 'const std::reverse_iterator<_BidIt> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\utility(484,27): message : or       'bool std::operator ==(const std::pair<_Ty1,_Ty2> &,const std::pair<_Ty1,_Ty2> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::pair<_Ty1,_Ty2> &,const std::pair<_Ty1,_Ty2> &)': could not deduce template argument for 'const std::pair<_Ty1,_Ty2> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\utility(484,27): message : or 'bool std::operator ==(const std::pair<_Ty1,_Ty2> &,const std::pair<_Ty1,_Ty2> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::pair<_Ty1,_Ty2> &,const std::pair<_Ty1,_Ty2> &)': could not deduce template argument for 'const std::pair<_Ty1,_Ty2> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xutility(1464,5): message : or 'bool std::operator ==(const std::reverse_iterator<_BidIt> &,const std::reverse_iterator<_BidIt2> &) noexcept(<expr>)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::reverse_iterator<_BidIt> &,const std::reverse_iterator<_BidIt2> &) noexcept(<expr>)': could not deduce template argument for 'const std::reverse_iterator<_BidIt> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xutility(4162,5): message : or 'bool std::operator ==(const std::move_iterator<_Iter> &,const std::move_iterator<_Iter2> &) noexcept(<expr>)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::move_iterator<_Iter> &,const std::move_iterator<_Iter2> &) noexcept(<expr>)': could not deduce template argument for 'const std::move_iterator<_Iter> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\tuple(669,27): message : or 'bool std::operator ==(const std::tuple<_Types...> &,const std::tuple<_Types...> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::tuple<_Types...> &,const std::tuple<_Types...> &)': could not deduce template argument for 'const std::tuple<_Types...> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xmemory(1011,30): message : or 'bool std::operator ==(const std::allocator<_Ty> &,const std::allocator<_Other> &) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::allocator<_Ty> &,const std::allocator<_Other> &) noexcept': could not deduce template argument for 'const std::allocator<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(1680,27): message : or 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const std::basic_string_view<_Elem,_Traits>) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const std::basic_string_view<_Elem,_Traits>) noexcept': could not deduce template argument for 'const std::basic_string_view<_Elem,_Traits>' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(1694,27): message : or 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const _Identity<std::basic_string_view<_Elem,_Traits>>::type) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const _Identity<std::basic_string_view<_Elem,_Traits>>::type) noexcept': could not deduce template argument for 'const std::basic_string_view<_Elem,_Traits>' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(5045,30): message : or 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const std::basic_string<_Elem,_Traits,_Alloc> &) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const std::basic_string<_Elem,_Traits,_Alloc> &) noexcept': could not deduce template argument for 'const std::basic_string<_Elem,_Traits,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(5051,30): message : or 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const _Elem *const )' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const _Elem *const )': could not deduce template argument for 'const std::basic_string<_Elem,_Traits,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\vector(2219,30): message : or 'bool std::operator ==(const std::vector<_Ty,_Alloc> &,const std::vector<_Ty,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::vector<_Ty,_Alloc> &,const std::vector<_Ty,_Alloc> &)': could not deduce template argument for 'const std::vector<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\list(1873,[17](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:18)): message : or 'bool std::operator ==(const std::list<_Ty,_Alloc> &,const std::list<_Ty,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::list<_Ty,_Alloc> &,const std::list<_Ty,_Alloc> &)': could not deduce template argument for 'const std::list<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\unordered_map(539,17): message : or 'bool std::operator ==(const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)': could not deduce template argument for 'const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\unordered_map(908,17): message : or 'bool std::operator ==(const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)': could not deduce template argument for 'const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\functional(1158,17): message : or 'bool std::operator ==(const std::function<_Fty> &,std::nullptr_t) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::function<_Fty> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::function<_Fty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(619,27): message : or 'bool std::operator ==(const std::optional<_Ty> &,const std::optional<_Ty2> &) noexcept(<expr>)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::optional<_Ty> &,const std::optional<_Ty2> &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(735,27): message : or 'bool std::operator ==(const std::optional<_Ty> &,std::nullopt_t) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::optional<_Ty> &,std::nullopt_t) noexcept': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(824,27): message : or 'bool std::operator ==(const std::optional<_Ty> &,const _Ty2 &) noexcept(<expr>)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::optional<_Ty> &,const _Ty2 &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(833,27): message : or 'bool std::operator ==(const _Ty1 &,const std::optional<_Ty2> &) noexcept(<expr>)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const _Ty1 &,const std::optional<_Ty2> &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty2> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\iterator(307,17): message : or 'bool std::operator ==(const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &,const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &,const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &) noexcept': could not deduce template argument for 'const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\iterator(489,17): message : or 'bool std::operator ==(const std::istreambuf_iterator<_Elem,_Traits> &,const std::istreambuf_iterator<_Elem,_Traits> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::istreambuf_iterator<_Elem,_Traits> &,const std::istreambuf_iterator<_Elem,_Traits> &)': could not deduce template argument for 'const std::istreambuf_iterator<_Elem,_Traits> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory([18](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:19)72,17): message : or 'bool std::operator ==(const std::shared_ptr<_Ty> &,const std::shared_ptr<_Ty0> &) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::shared_ptr<_Ty> &,const std::shared_ptr<_Ty0> &) noexcept': could not deduce template argument for 'const std::shared_ptr<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory([19](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:20)09,17): message : or 'bool std::operator ==(const std::shared_ptr<_Ty> &,std::nullptr_t) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::shared_ptr<_Ty> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::shared_ptr<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\[20](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:21)22\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(3501,30): message : or 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,const std::unique_ptr<_Ty2,_Dx2> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,const std::unique_ptr<_Ty2,_Dx2> &)': could not deduce template argument for 'const std::unique_ptr<_Ty,_Dx> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(3549,30): message : or 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,std::nullptr_t) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::unique_ptr<_Ty,_Dx> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\sharedptr.h(158,6): message : or 'bool operator ==(const wxSharedPtr<T> &,const wxSharedPtr<U> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool operator ==(const wxSharedPtr<T> &,const wxSharedPtr<U> &)': could not deduce template argument for 'const wxSharedPtr<T> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\array(771,30): message : or 'bool std::operator ==(const std::array<_Ty,_Size> &,const std::array<_Ty,_Size> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::array<_Ty,_Size> &,const std::array<_Ty,_Size> &)': could not deduce template argument for 'const std::array<_Ty,_Size> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\map(399,17): message : or 'bool std::operator ==(const std::map<_Kty,_Ty,_Pr,_Alloc> &,const std::map<_Kty,_Ty,_Pr,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::map<_Kty,_Ty,_Pr,_Alloc> &,const std::map<_Kty,_Ty,_Pr,_Alloc> &)': could not deduce template argument for 'const std::map<_Kty,_Ty,_Pr,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\map(630,17): message : or 'bool std::operator ==(const std::multimap<_Kty,_Ty,_Pr,_Alloc> &,const std::multimap<_Kty,_Ty,_Pr,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::multimap<_Kty,_Ty,_Pr,_Alloc> &,const std::multimap<_Kty,_Ty,_Pr,_Alloc> &)': could not deduce template argument for 'const std::multimap<_Kty,_Ty,_Pr,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\deque(1703,17): message : or 'bool std::operator ==(const std::deque<_Ty,_Alloc> &,const std::deque<_Ty,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::deque<_Ty,_Alloc> &,const std::deque<_Ty,_Alloc> &)': could not deduce template argument for 'const std::deque<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\queue(181,17): message : or 'bool std::operator ==(const std::queue<_Ty,_Container> &,const std::queue<_Ty,_Container> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::queue<_Ty,_Container> &,const std::queue<_Ty,_Container> &)': could not deduce template argument for 'const std::queue<_Ty,_Container> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\forward_list(1570,17): message : or 'bool std::operator ==(const std::forward_list<_Ty,_Alloc> &,const std::forward_list<_Ty,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : 'bool std::operator ==(const std::forward_list<_Ty,_Alloc> &,const std::forward_list<_Ty,_Alloc> &)': could not deduce template argument for 'const std::forward_list<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(350,15): message : while trying to match the argument list '(treesheets::Selection, treesheets::Selection)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): error C2666: 'treesheets::Selection::operator ==': overloaded functions have similar conversions [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\selection.h(34,10): message : could be 'bool treesheets::Selection::operator ==(const treesheets::Selection &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\pen.h(138,13): message : or       'bool operator ==(wxPenStyle,wxDeprecatedGUIConstants)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\brush.h(107,13): message : or       'bool operator ==(wxBrushStyle,wxDeprecatedGUIConstants)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(138,13): message : or       'bool operator ==(long,const wxWindowIDRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(133,13): message : or       'bool operator ==(int,const wxWindowIDRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(128,13): message : or       'bool operator ==(const wxWindowIDRef &,long)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(123,13): message : or       'bool operator ==(const wxWindowIDRef &,int)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(118,13): message : or       'bool operator ==(const wxWindowIDRef &,const wxWindowIDRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(688,13): message : or       'bool operator ==(wxFontWeight,wxDeprecatedGUIConstants)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(682,13): message : or       'bool operator ==(wxFontStyle,wxDeprecatedGUIConstants)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(676,13): message : or       'bool operator ==(wxFontFamily,wxDeprecatedGUIConstants)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.226[21](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:22).0\shared\guiddef.h(192,15): message : or       'bool operator ==(const GUID &,const GUID &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(878,13): message : or       'bool operator ==(const wxRect &,const wxRect &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(602,13): message : or       'bool operator ==(const wxPoint &,const wxPoint &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(472,13): message : or       'bool operator ==(const wxRealPoint &,const wxRealPoint &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(350,13): message : or       'bool operator ==(const wxSize &,const wxSize &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\longlong.h(1056,13): message : or       'bool operator ==(unsigned long,const wxULongLong &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\longlong.h(1043,13): message : or       'bool operator ==(long,const wxLongLong &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4084,1): message : or       'bool operator ==(const wxCStrData &,const char *)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4084,1): message : or       'bool operator ==(const char *,const wxCStrData &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4082,1): message : or       'bool operator ==(const wxCStrData &,const wchar_t *)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4082,1): message : or       'bool operator ==(const wchar_t *,const wxCStrData &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4035,13): message : or       'bool operator ==(const wxString &,wchar_t)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4034,13): message : or       'bool operator ==(const wxString &,char)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4033,13): message : or       'bool operator ==(const wxString &,const wxUniCharRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4032,13): message : or       'bool operator ==(const wxString &,const wxUniChar &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4031,13): message : or       'bool operator ==(int,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4030,13): message : or       'bool operator ==(wchar_t,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4029,13): message : or       'bool operator ==(char,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4028,13): message : or       'bool operator ==(const wxUniCharRef &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4027,13): message : or       'bool operator ==(const wxUniChar &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4006,13): message : or       'bool operator ==(const wxScopedCharBuffer &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4004,13): message : or       'bool operator ==(const wxString &,const wxScopedCharBuffer &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3996,13): message : or       'bool operator ==(const wxScopedWCharBuffer &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3994,13): message : or       'bool operator ==(const wxString &,const wxScopedWCharBuffer &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3987,13): message : or       'bool operator ==(const wxCStrData &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3985,13): message : or       'bool operator ==(const wxString &,const wxCStrData &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3972,13): message : or       'bool operator ==(const wxString &,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3967,1): message : or       'bool operator ==(const wxString &,const char *)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3967,1): message : or       'bool operator ==(const char *,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3965,1): message : or       'bool operator ==(const wxString &,const wchar_t *)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3965,1): message : or       'bool operator ==(const wchar_t *,const wxString &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(378,1): message : or       'bool operator ==(const wxUniChar &,const wxUniCharRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(376,1): message : or       'bool operator ==(wchar_t,const wxUniCharRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(375,1): message : or       'bool operator ==(wchar_t,const wxUniChar &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(373,1): message : or       'bool operator ==(char,const wxUniCharRef &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(372,1): message : or       'bool operator ==(char,const wxUniChar &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\20[22](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:23)\Enterprise\VC\Tools\MSVC\14.36.32532\include\thread(242,36): message : or       'bool std::operator ==(std::thread::id,std::thread::id) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\selection.h(34,10): message : or 'bool treesheets::Selection::operator ==(const treesheets::Selection &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(372,1): message : or 'bool operator ==(char,const wxUniChar &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(373,1): message : or 'bool operator ==(char,const wxUniCharRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(375,1): message : or 'bool operator ==(wchar_t,const wxUniChar &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(376,1): message : or 'bool operator ==(wchar_t,const wxUniCharRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\unichar.h(378,1): message : or 'bool operator ==(const wxUniChar &,const wxUniCharRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3965,1): message : or 'bool operator ==(const wchar_t *,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3965,1): message : or 'bool operator ==(const wxString &,const wchar_t *)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3967,1): message : or 'bool operator ==(const char *,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3967,1): message : or 'bool operator ==(const wxString &,const char *)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3972,13): message : or 'bool operator ==(const wxString &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3985,13): message : or 'bool operator ==(const wxString &,const wxCStrData &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3987,13): message : or 'bool operator ==(const wxCStrData &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3994,13): message : or 'bool operator ==(const wxString &,const wxScopedWCharBuffer &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(3996,13): message : or 'bool operator ==(const wxScopedWCharBuffer &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4004,13): message : or 'bool operator ==(const wxString &,const wxScopedCharBuffer &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4006,13): message : or 'bool operator ==(const wxScopedCharBuffer &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4027,13): message : or 'bool operator ==(const wxUniChar &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4028,13): message : or 'bool operator ==(const wxUniCharRef &,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4029,13): message : or 'bool operator ==(char,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4030,13): message : or 'bool operator ==(wchar_t,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4031,13): message : or 'bool operator ==(int,const wxString &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4032,13): message : or 'bool operator ==(const wxString &,const wxUniChar &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4033,13): message : or 'bool operator ==(const wxString &,const wxUniCharRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4034,13): message : or 'bool operator ==(const wxString &,char)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4035,13): message : or 'bool operator ==(const wxString &,wchar_t)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4082,1): message : or 'bool operator ==(const wchar_t *,const wxCStrData &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4082,1): message : or 'bool operator ==(const wxCStrData &,const wchar_t *)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4084,1): message : or 'bool operator ==(const char *,const wxCStrData &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\string.h(4084,1): message : or 'bool operator ==(const wxCStrData &,const char *)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\longlong.h(1043,13): message : or 'bool operator ==(long,const wxLongLong &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\longlong.h(1056,13): message : or 'bool operator ==(unsigned long,const wxULongLong &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(350,13): message : or 'bool operator ==(const wxSize &,const wxSize &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(472,13): message : or 'bool operator ==(const wxRealPoint &,const wxRealPoint &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(602,13): message : or 'bool operator ==(const wxPoint &,const wxPoint &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\gdicmn.h(878,13): message : or 'bool operator ==(const wxRect &,const wxRect &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared\guiddef.h(192,15): message : or 'bool operator ==(const GUID &,const GUID &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(676,13): message : or 'bool operator ==(wxFontFamily,wxDeprecatedGUIConstants)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(682,13): message : or 'bool operator ==(wxFontStyle,wxDeprecatedGUIConstants)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\font.h(688,13): message : or 'bool operator ==(wxFontWeight,wxDeprecatedGUIConstants)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(118,13): message : or 'bool operator ==(const wxWindowIDRef &,const wxWindowIDRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(1[23](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:24),13): message : or 'bool operator ==(const wxWindowIDRef &,int)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(128,13): message : or 'bool operator ==(const wxWindowIDRef &,long)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(133,13): message : or 'bool operator ==(int,const wxWindowIDRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\windowid.h(138,13): message : or 'bool operator ==(long,const wxWindowIDRef &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\brush.h(107,13): message : or 'bool operator ==(wxBrushStyle,wxDeprecatedGUIConstants)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\pen.h(138,13): message : or 'bool operator ==(wxPenStyle,wxDeprecatedGUIConstants)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\thread([24](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:25)2,36): message : or 'bool std::operator ==(std::thread::id,std::thread::id) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\sharedptr.h(158,6): message : or       'bool operator ==(const wxSharedPtr<T> &,const wxSharedPtr<U> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool operator ==(const wxSharedPtr<T> &,const wxSharedPtr<U> &)': could not deduce template argument for 'const wxSharedPtr<T> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.3[25](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:26)32\include\forward_list(1570,17): message : or       'bool std::operator ==(const std::forward_list<_Ty,_Alloc> &,const std::forward_list<_Ty,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::forward_list<_Ty,_Alloc> &,const std::forward_list<_Ty,_Alloc> &)': could not deduce template argument for 'const std::forward_list<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\queue(181,17): message : or       'bool std::operator ==(const std::queue<_Ty,_Container> &,const std::queue<_Ty,_Container> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::queue<_Ty,_Container> &,const std::queue<_Ty,_Container> &)': could not deduce template argument for 'const std::queue<_Ty,_Container> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\deque(1703,17): message : or       'bool std::operator ==(const std::deque<_Ty,_Alloc> &,const std::deque<_Ty,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::deque<_Ty,_Alloc> &,const std::deque<_Ty,_Alloc> &)': could not deduce template argument for 'const std::deque<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\map(630,17): message : or       'bool std::operator ==(const std::multimap<_Kty,_Ty,_Pr,_Alloc> &,const std::multimap<_Kty,_Ty,_Pr,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::multimap<_Kty,_Ty,_Pr,_Alloc> &,const std::multimap<_Kty,_Ty,_Pr,_Alloc> &)': could not deduce template argument for 'const std::multimap<_Kty,_Ty,_Pr,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\map(399,17): message : or       'bool std::operator ==(const std::map<_Kty,_Ty,_Pr,_Alloc> &,const std::map<_Kty,_Ty,_Pr,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::map<_Kty,_Ty,_Pr,_Alloc> &,const std::map<_Kty,_Ty,_Pr,_Alloc> &)': could not deduce template argument for 'const std::map<_Kty,_Ty,_Pr,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\array(771,30): message : or       'bool std::operator ==(const std::array<_Ty,_Size> &,const std::array<_Ty,_Size> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::array<_Ty,_Size> &,const std::array<_Ty,_Size> &)': could not deduce template argument for 'const std::array<_Ty,_Size> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(3549,30): message : or       'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,std::nullptr_t) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::unique_ptr<_Ty,_Dx> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(3501,30): message : or       'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,const std::unique_ptr<_Ty2,_Dx2> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,const std::unique_ptr<_Ty2,_Dx2> &)': could not deduce template argument for 'const std::unique_ptr<_Ty,_Dx> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(1909,17): message : or       'bool std::operator ==(const std::shared_ptr<_Ty> &,std::nullptr_t) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::shared_ptr<_Ty> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::shared_ptr<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(1872,17): message : or       'bool std::operator ==(const std::shared_ptr<_Ty> &,const std::shared_ptr<_Ty0> &) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::shared_ptr<_Ty> &,const std::shared_ptr<_Ty0> &) noexcept': could not deduce template argument for 'const std::shared_ptr<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\iterator(489,17): message : or       'bool std::operator ==(const std::istreambuf_iterator<_Elem,_Traits> &,const std::istreambuf_iterator<_Elem,_Traits> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::istreambuf_iterator<_Elem,_Traits> &,const std::istreambuf_iterator<_Elem,_Traits> &)': could not deduce template argument for 'const std::istreambuf_iterator<_Elem,_Traits> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\iterator(307,17): message : or       'bool std::operator ==(const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &,const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &,const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &) noexcept': could not deduce template argument for 'const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(833,27): message : or       'bool std::operator ==(const _Ty1 &,const std::optional<_Ty2> &) noexcept(<expr>)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const _Ty1 &,const std::optional<_Ty2> &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty2> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(824,27): message : or       'bool std::operator ==(const std::optional<_Ty> &,const _Ty2 &) noexcept(<expr>)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::optional<_Ty> &,const _Ty2 &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(735,27): message : or       'bool std::operator ==(const std::optional<_Ty> &,std::nullopt_t) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::optional<_Ty> &,std::nullopt_t) noexcept': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(619,27): message : or       'bool std::operator ==(const std::optional<_Ty> &,const std::optional<_Ty2> &) noexcept(<expr>)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::optional<_Ty> &,const std::optional<_Ty2> &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\functional(1158,17): message : or       'bool std::operator ==(const std::function<_Fty> &,std::nullptr_t) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::function<_Fty> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::function<_Fty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\unordered_map(908,17): message : or       'bool std::operator ==(const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)': could not deduce template argument for 'const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\unordered_map(539,17): message : or       'bool std::operator ==(const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)': could not deduce template argument for 'const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\list(1873,17): message : or       'bool std::operator ==(const std::list<_Ty,_Alloc> &,const std::list<_Ty,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::list<_Ty,_Alloc> &,const std::list<_Ty,_Alloc> &)': could not deduce template argument for 'const std::list<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\vector(2219,30): message : or       'bool std::operator ==(const std::vector<_Ty,_Alloc> &,const std::vector<_Ty,_Alloc> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::vector<_Ty,_Alloc> &,const std::vector<_Ty,_Alloc> &)': could not deduce template argument for 'const std::vector<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(5051,30): message : or       'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const _Elem *const )' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const _Elem *const )': could not deduce template argument for 'const std::basic_string<_Elem,_Traits,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(5045,30): message : or       'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const std::basic_string<_Elem,_Traits,_Alloc> &) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const std::basic_string<_Elem,_Traits,_Alloc> &) noexcept': could not deduce template argument for 'const std::basic_string<_Elem,_Traits,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(1694,27): message : or       'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const _Identity<std::basic_string_view<_Elem,_Traits>>::type) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const _Identity<std::basic_string_view<_Elem,_Traits>>::type) noexcept': could not deduce template argument for 'const std::basic_string_view<_Elem,_Traits>' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(1680,27): message : or       'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const std::basic_string_view<_Elem,_Traits>) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const std::basic_string_view<_Elem,_Traits>) noexcept': could not deduce template argument for 'const std::basic_string_view<_Elem,_Traits>' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xmemory(1011,30): message : or       'bool std::operator ==(const std::allocator<_Ty> &,const std::allocator<_Other> &) noexcept' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::allocator<_Ty> &,const std::allocator<_Other> &) noexcept': could not deduce template argument for 'const std::allocator<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\tuple(669,27): message : or       'bool std::operator ==(const std::tuple<_Types...> &,const std::tuple<_Types...> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::tuple<_Types...> &,const std::tuple<_Types...> &)': could not deduce template argument for 'const std::tuple<_Types...> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xutility(4162,5): message : or       'bool std::operator ==(const std::move_iterator<_Iter> &,const std::move_iterator<_Iter2> &) noexcept(<expr>)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::move_iterator<_Iter> &,const std::move_iterator<_Iter2> &) noexcept(<expr>)': could not deduce template argument for 'const std::move_iterator<_Iter> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xutility(1464,5): message : or       'bool std::operator ==(const std::reverse_iterator<_BidIt> &,const std::reverse_iterator<_BidIt2> &) noexcept(<expr>)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::reverse_iterator<_BidIt> &,const std::reverse_iterator<_BidIt2> &) noexcept(<expr>)': could not deduce template argument for 'const std::reverse_iterator<_BidIt> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\utility(484,27): message : or       'bool std::operator ==(const std::pair<_Ty1,_Ty2> &,const std::pair<_Ty1,_Ty2> &)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::pair<_Ty1,_Ty2> &,const std::pair<_Ty1,_Ty2> &)': could not deduce template argument for 'const std::pair<_Ty1,_Ty2> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\utility(484,27): message : or 'bool std::operator ==(const std::pair<_Ty1,_Ty2> &,const std::pair<_Ty1,_Ty2> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::pair<_Ty1,_Ty2> &,const std::pair<_Ty1,_Ty2> &)': could not deduce template argument for 'const std::pair<_Ty1,_Ty2> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xutility(1464,5): message : or 'bool std::operator ==(const std::reverse_iterator<_BidIt> &,const std::reverse_iterator<_BidIt2> &) noexcept(<expr>)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::reverse_iterator<_BidIt> &,const std::reverse_iterator<_BidIt2> &) noexcept(<expr>)': could not deduce template argument for 'const std::reverse_iterator<_BidIt> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xutility(4162,5): message : or 'bool std::operator ==(const std::move_iterator<_Iter> &,const std::move_iterator<_Iter2> &) noexcept(<expr>)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::move_iterator<_Iter> &,const std::move_iterator<_Iter2> &) noexcept(<expr>)': could not deduce template argument for 'const std::move_iterator<_Iter> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\tuple(669,27): message : or 'bool std::operator ==(const std::tuple<_Types...> &,const std::tuple<_Types...> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::tuple<_Types...> &,const std::tuple<_Types...> &)': could not deduce template argument for 'const std::tuple<_Types...> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xmemory(1011,30): message : or 'bool std::operator ==(const std::allocator<_Ty> &,const std::allocator<_Other> &) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::allocator<_Ty> &,const std::allocator<_Other> &) noexcept': could not deduce template argument for 'const std::allocator<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(1680,27): message : or 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const std::basic_string_view<_Elem,_Traits>) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const std::basic_string_view<_Elem,_Traits>) noexcept': could not deduce template argument for 'const std::basic_string_view<_Elem,_Traits>' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(1694,27): message : or 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const _Identity<std::basic_string_view<_Elem,_Traits>>::type) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::basic_string_view<_Elem,_Traits>,const _Identity<std::basic_string_view<_Elem,_Traits>>::type) noexcept': could not deduce template argument for 'const std::basic_string_view<_Elem,_Traits>' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(5045,30): message : or 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const std::basic_string<_Elem,_Traits,_Alloc> &) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const std::basic_string<_Elem,_Traits,_Alloc> &) noexcept': could not deduce template argument for 'const std::basic_string<_Elem,_Traits,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\xstring(5051,30): message : or 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const _Elem *const )' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::basic_string<_Elem,_Traits,_Alloc> &,const _Elem *const )': could not deduce template argument for 'const std::basic_string<_Elem,_Traits,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\vector(2219,30): message : or 'bool std::operator ==(const std::vector<_Ty,_Alloc> &,const std::vector<_Ty,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::vector<_Ty,_Alloc> &,const std::vector<_Ty,_Alloc> &)': could not deduce template argument for 'const std::vector<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\list(1873,17): message : or 'bool std::operator ==(const std::list<_Ty,_Alloc> &,const std::list<_Ty,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::list<_Ty,_Alloc> &,const std::list<_Ty,_Alloc> &)': could not deduce template argument for 'const std::list<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\unordered_map(539,17): message : or 'bool std::operator ==(const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)': could not deduce template argument for 'const std::unordered_map<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\unordered_map(908,17): message : or 'bool std::operator ==(const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &,const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &)': could not deduce template argument for 'const std::unordered_multimap<_Kty,_Ty,_Hasher,_Keyeq,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\functional(1158,17): message : or 'bool std::operator ==(const std::function<_Fty> &,std::nullptr_t) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::function<_Fty> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::function<_Fty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(619,27): message : or 'bool std::operator ==(const std::optional<_Ty> &,const std::optional<_Ty2> &) noexcept(<expr>)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::optional<_Ty> &,const std::optional<_Ty2> &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(735,27): message : or 'bool std::operator ==(const std::optional<_Ty> &,std::nullopt_t) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::optional<_Ty> &,std::nullopt_t) noexcept': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(824,27): message : or 'bool std::operator ==(const std::optional<_Ty> &,const _Ty2 &) noexcept(<expr>)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::optional<_Ty> &,const _Ty2 &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\optional(833,27): message : or 'bool std::operator ==(const _Ty1 &,const std::optional<_Ty2> &) noexcept(<expr>)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const _Ty1 &,const std::optional<_Ty2> &) noexcept(<expr>)': could not deduce template argument for 'const std::optional<_Ty2> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\iterator(307,17): message : or 'bool std::operator ==(const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &,const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &,const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &) noexcept': could not deduce template argument for 'const std::istream_iterator<_Ty,_Elem,_Traits,_Diff> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\iterator(489,17): message : or 'bool std::operator ==(const std::istreambuf_iterator<_Elem,_Traits> &,const std::istreambuf_iterator<_Elem,_Traits> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::istreambuf_iterator<_Elem,_Traits> &,const std::istreambuf_iterator<_Elem,_Traits> &)': could not deduce template argument for 'const std::istreambuf_iterator<_Elem,_Traits> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(1872,17): message : or 'bool std::operator ==(const std::shared_ptr<_Ty> &,const std::shared_ptr<_Ty0> &) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::shared_ptr<_Ty> &,const std::shared_ptr<_Ty0> &) noexcept': could not deduce template argument for 'const std::shared_ptr<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(1909,17): message : or 'bool std::operator ==(const std::shared_ptr<_Ty> &,std::nullptr_t) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::shared_ptr<_Ty> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::shared_ptr<_Ty> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(3501,30): message : or 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,const std::unique_ptr<_Ty2,_Dx2> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,const std::unique_ptr<_Ty2,_Dx2> &)': could not deduce template argument for 'const std::unique_ptr<_Ty,_Dx> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\memory(3549,30): message : or 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,std::nullptr_t) noexcept' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::unique_ptr<_Ty,_Dx> &,std::nullptr_t) noexcept': could not deduce template argument for 'const std::unique_ptr<_Ty,_Dx> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\wxWidgets\include\wx\sharedptr.h(158,6): message : or 'bool operator ==(const wxSharedPtr<T> &,const wxSharedPtr<U> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool operator ==(const wxSharedPtr<T> &,const wxSharedPtr<U> &)': could not deduce template argument for 'const wxSharedPtr<T> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\array(771,30): message : or 'bool std::operator ==(const std::array<_Ty,_Size> &,const std::array<_Ty,_Size> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::array<_Ty,_Size> &,const std::array<_Ty,_Size> &)': could not deduce template argument for 'const std::array<_Ty,_Size> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\map(399,17): message : or 'bool std::operator ==(const std::map<_Kty,_Ty,_Pr,_Alloc> &,const std::map<_Kty,_Ty,_Pr,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::map<_Kty,_Ty,_Pr,_Alloc> &,const std::map<_Kty,_Ty,_Pr,_Alloc> &)': could not deduce template argument for 'const std::map<_Kty,_Ty,_Pr,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\map(630,17): message : or 'bool std::operator ==(const std::multimap<_Kty,_Ty,_Pr,_Alloc> &,const std::multimap<_Kty,_Ty,_Pr,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::multimap<_Kty,_Ty,_Pr,_Alloc> &,const std::multimap<_Kty,_Ty,_Pr,_Alloc> &)': could not deduce template argument for 'const std::multimap<_Kty,_Ty,_Pr,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\deque(1703,17): message : or 'bool std::operator ==(const std::deque<_Ty,_Alloc> &,const std::deque<_Ty,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::deque<_Ty,_Alloc> &,const std::deque<_Ty,_Alloc> &)': could not deduce template argument for 'const std::deque<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\queue(181,17): message : or 'bool std::operator ==(const std::queue<_Ty,_Container> &,const std::queue<_Ty,_Container> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::queue<_Ty,_Container> &,const std::queue<_Ty,_Container> &)': could not deduce template argument for 'const std::queue<_Ty,_Container> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\include\forward_list(1570,17): message : or 'bool std::operator ==(const std::forward_list<_Ty,_Alloc> &,const std::forward_list<_Ty,_Alloc> &)' [synthesized expression 'y == x'] [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : 'bool std::operator ==(const std::forward_list<_Ty,_Alloc> &,const std::forward_list<_Ty,_Alloc> &)': could not deduce template argument for 'const std::forward_list<_Ty,_Alloc> &' from 'treesheets::Selection' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
D:\a\treesheets\treesheets\src\document.h(495,19): message : while trying to match the argument list '(treesheets::Selection, treesheets::Selection)' [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
Done Building Project "D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj" (default targets) -- FAILED.
Done Building Project "D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj.metaproj" (default targets) -- FAILED.
Done Building Project "D:\a\treesheets\treesheets\TreeSheets\treesheets.sln" (default targets) -- FAILED.

Build FAILED.

"D:\a\treesheets\treesheets\TreeSheets\treesheets.sln" (default target) (1) ->
"D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj.metaproj" (default target) (2) ->
"D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj" (default target) (4) ->
(ClCompile target) -> 
  D:\a\treesheets\treesheets\src\document.h(350,15): error C[26](https://github.com/t2b3/treesheets/actions/runs/5337792015/jobs/9674286960#step:6:27)66: 'treesheets::Selection::operator ==': overloaded functions have similar conversions [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]
  D:\a\treesheets\treesheets\src\document.h(495,19): error C2666: 'treesheets::Selection::operator ==': overloaded functions have similar conversions [D:\a\treesheets\treesheets\TreeSheets\treesheets.vcxproj]

    0 Warning(s)
    2 Error(s)
```

Reply:
Wow, that's a very confused error.. I guess they may have a bug in their compiler since I don't see anything wrong with the use of `==`.

Reply:
> 
I absolutely agree! This is pretty crazy. Hopefully it will be fixed soon.